### PR TITLE
Add request id to error messages

### PR DESCRIPTION
**Pull request by schirevko**

### CODE REVIEW

File: vkcs/internal/util/util.go [hunk 1]
```diff
@@ -18,6 +18,8 @@ import (
 	"github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/clients"
 )
 
+const RequestIDHeader = "X-Openstack-Request-Id"
```
Comment (ftersin) [inline vkcs/internal/util/util.go:21]: does it work with karboii and ghosts?

Comment (schirevko) [reply vkcs/internal/util/util.go:22]: These services do not send any request id, thus we do not handle it in corresponding resources

File: vkcs/internal/diagnostic/diagnostic.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package diagnostic
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+)
+
+type DiagnosticWithRequestID struct {
```
Comment (paaanic) [inline vkcs/internal/diagnostic/diagnostic.go:9]: Let's rename this file to `request_id.go`.

**Review by paaanic — CHANGES_REQUESTED** [review]

File: vkcs/internal/diagnostic/diagnostic.go (hunk 2 above)
Comment (schirevko) [reply vkcs/internal/diagnostic/diagnostic.go]: This file is no longer necessary, removed it

**Comment by paaanic** [comment]
May be we should somehow handle situation when Request ID header is absent, for now output is: "Request ID: ". Wouldn't it be better to omit it at all?

**Review by paaanic — APPROVED** [review]

**Pull request merged**